web/server/entrypoint.go

```diff
@@ -261,6 +262,8 @@ func (s *robotServer) serveWeb(ctx context.Context, cfg *config.Config) (err err
 	forceShutdown := make(chan struct{})
 	defer func() { <-forceShutdown }()
 
+	var myRobot robot.LocalRobot
+
 	utils.PanicCapturingGo(func() {
```

Comment:
Would be awesome if this force shutdown goroutine was its own method/function as we've been doing for our various unnamed async lambdas.